fig/python-zero-index.svg

```diff
@@ -1,14 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- Generator: Adobe Illustrator 21.0.2, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
 <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-	 viewBox="0 0 460 74.6" style="enable-background:new 0 0 460 74.6;" xml:space="preserve">
+	 viewBox="0 0 460 74.6" style="enable-background:new 0 0 460 74.6;" xml:space="preserve"
+	 width="2000px" height="326px">
 <style type="text/css">
 	.st0{opacity:0.75;fill:#FFF200;}
 	.st1{opacity:0.5;fill:#F7941D;}
 	.st2{opacity:0.25;fill:#EC008C;}
 	.st3{fill:none;}
 	.st4{fill:#00A651;}
-	.st5{font-family:'Helvetica-Bold';}
+	.st5{font-family:'sans-serif';}
```

Comment:
Perhaps the `font-weight="bold"` should be in here?

Reply:
I tried to place it there (e.g., `.st5{font-family:'sans-serif';font-weight:'bold';}`) and remove all the calls I was making to `font-weight=bold`, but none of the text was becoming bold...I'm not sure what I'm missing.

Reply:
When I try it without the quotes around bold it works for me. Does `.st5{font-family:'sans-serif';font-weight:bold;}` work for you?

Reply:
you can also use numeric values: 400 for normal, 700 for bold.